### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by anik314159**

### CODE REVIEW

File: config/loopdist/Config.h
Comment (svkeerthy) [inline config/loopdist/Config.h:1]: Why is this file needed?

**Review by svkeerthy — CHANGES_REQUESTED** [review]

File: config/loopdist/Config.h
Comment (anik314159) [reply config/loopdist/Config.h:1]: Deleted the file,

**Review by svkeerthy — APPROVED** [review]
